pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.3
+lockfileVersion: 5.4
```

Comment:
we're not yet using pnpm v8 in this repo (see: https://github.com/rollup/plugins/blob/0f73e2ace5774b5b3e78584c6d1ed90d204a319e/package.json#L41) please downgrade temporarily, reset the pnpm-lock.yaml file and run install again for this PR.

(or we can wait on this PR until we update the pnpm usage in the repo)

Reply:
Hi, it's strange that I already used pnpm@7.12.2, but after I run `pnpm i`, it still upgrade to lock file 5.4.

Reply:
ok, I updated, it looks good now.